.github/workflows/build.yml

```diff
@@ -231,13 +231,17 @@ jobs:
     name: >-
       Ubuntu
       ${{ fromJSON(matrix.free-threading) && '(free-threading)' || '' }}
+      ${{ fromJSON(matrix.bolt) && '(bolt)' || '' }}
     needs: check_source
     if: needs.check_source.outputs.run_tests == 'true'
     strategy:
       matrix:
         free-threading:
         - false
         - true
+        bolt:
+        - false
+        - true
```

Comment:
Nit-picking, put `bolt` alphabetically before `free-threading`, to match the matrix view?

<img width="289" alt="image" src="https://github.com/user-attachments/assets/4f3391ae-3cc4-4c3a-be72-e600547dc815" />


```suggestion
        bolt:
        - false
        - true
        free-threading:
        - false
        - true
```